### DISCUSSION TITLE
[Projects Beta] Search is unable to find repository

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [community/community#9198] [Projects and Issues] [answered]
I have the https://github.com/PythonCoderAS/watch-gh-repos repository, but it does not show up when I search for it in the repository column.

**Comment by PythonCoderAS** [comment, marked as answer]
So it turns out issues were disabled for whatever unknown reason. The original fork had issues enabled so this is strange.

**Reply by Vijaysaini1** [reply]
i am quite will of here and how are
